style.css

```diff
@@ -1,15 +1,34 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin: 0;
+}
+main {
+  max-width: 350px;
+  margin-left: auto;
+  margin-right: auto;
+}
 h1 {
     font-size: 100px;
 }
-
-a img {
-    border:none;
-    outline:none;
+h2 {
+    text-align: center;
+}
+img {
+    max-width: 100%;
+}
+.social-icon {
+    text-decoration: none;
+}
+.social-icon {
```

Comment:
I also mentioned this in the comment in the main thread, but how would we go about making it so that I could easily add more icons and they'd automatically scale and fit? Is that possible? Right now, the code seems built for 3. Would I need to change it up every time I added one?